conda_forge_tick/auto_tick.xsh

```diff
@@ -88,20 +95,21 @@ def main():
                 rerender = (gx.nodes[node].get('smithy_version') != smithy_version or
                             gx.nodes[node].get('pinning_version') != pinning_version or
                             migrator.rerender)
-                run(attrs=attrs, migrator=migrator, gh=gh, rerender=rerender, protocol='https',
-                    hash_type=attrs.get('hash_type', 'sha256'))
+                migrator_hash = run(attrs=attrs, migrator=migrator, gh=gh,
+                                    rerender=rerender, protocol='https',
+                                    hash_type=attrs.get('hash_type', 'sha256'))
                 # TODO: capture pinning here too!
-                gx.nodes[node].update({'PRed': attrs['new_version'],
-                                       'smithy_version': smithy_version,
+                if 'PRed' not in gx.nodes[node]:
+                    gx.nodes[node]['PRed'] = set()
+                gx.nodes[node]['PRed'].add(migrator_hash)
```

Comment:
Maybe do `gx.nodes[node].setdefault('PRed', set()).add(migrator_hash)` here to reduce lines.